docs/en/reference/advanced-configuration.rst

```diff
@@ -71,9 +76,26 @@ Configuration Options
 The following sections describe all the configuration options
 available on a ``Doctrine\ORM\Configuration`` instance.
 
+Native Lazy Objects (***OPTIONAL***)
```

Comment:
```suggestion
Native Lazy Objects (**OPTIONAL**)
```

Reply:
https://github.com/doctrine/orm/pull/11885

Reply:
Or maybe that should be done only once https://github.com/doctrine/orm/pull/11885 is merged up, IDK